core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -2369,6 +2368,111 @@ public static NsfwLevel of(final int value) {
         }
     }
 
+    /**
+     * Describes the features of a guild.
+     * <br>
+     * You can see the available
+     * @see <a href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features">Guild Features</a>
+     */
+    public enum FeatureSet {
```

Comment:
I think you missed my point 😓 I meant either use a EnumSet<Feature> or use a custom FeatureSet class that works like the PermissionSet one

Reply:
In that case, a EnumSet is fine, so you can just rename the enum to `GuildFeature` for instance